packages/jsii-pacmak/test/dependency-graph.test.ts

```diff
@@ -113,3 +113,41 @@ test('stops traversing when callback returns false', async () => {
   expect(mockHost.readJson).toHaveBeenCalledTimes(2);
   expect(mockHost.findDependencyDirectory).toHaveBeenCalledTimes(1);
 });
+
+test('dont call findDependencyDirectory for bundledDependencies', async () => {
```

Comment:
Can you add flavors of this test for:
- the alternate spelling
- when both are used